src/Reflection/ReflectionHelper.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace NunoMaduro\Larastan\Reflection;
+
+use PHPStan\PhpDoc\Tag\PropertyTag;
+use PHPStan\Reflection\ClassReflection;
+
+final class ReflectionHelper
+{
+    /**
+     * Returns all property tags of the given class and its ancestors.
+     *
+     * In case of duplicates, the tags of the class itself have precedence.
+     * TODO consider the hierarchy to ensure correct precedence between ancestors
```

Comment:
Are there existing facilities in PHPStan/Larastan to do this?

If not, I would rather not implement this perfectly right now. It seems pretty hard to get right. Don't let perfect be the enemy of improvement 🤞